pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -2275,7 +2276,10 @@ func TestMultipleServiceChanges(t *testing.T) {
 	*controller = *newController(testServer.URL, 0*time.Second)
 	addPods(controller.podStore, ns, 1, 1, 0, ipv4only)
 
-	go func() { controller.Run(context.TODO(), 1) }()
+	ctx, cancel := context.WithCancel(context.TODO())
```

Comment:
I think we can remove the stopChan in line 2272

Reply:
+1